docs/docs/dev/policies/service-health-probes.md

```diff
@@ -127,21 +160,17 @@ networking:
         timeout: 2s # optional (default value is taken from KUMA_DP_SERVER_HDS_CHECK_TIMEOUT)
         interval: 1s # optional (default value is taken from KUMA_DP_SERVER_HDS_CHECK_INTERVAL)
         healthyThreshold: 1 # optional (default value is taken from KUMA_DP_SERVER_HDS_CHECK_HEALTHY_THRESHOLD)
-        unhealthThreshold: 1 # optional (default value is taken from KUMA_DP_SERVER_HDS_CHECK_UNHEALTHY_THRESHOLD)
+        unhealthyThreshold: 1 # optional (default value is taken from KUMA_DP_SERVER_HDS_CHECK_UNHEALTHY_THRESHOLD)
         tcp: {}
       tags:
         kuma.io/service: backend
         kuma.io/protocol: http
 ```
 
-If there is a `serviceProbe` configured for the inbound, Kuma will automatically fill the `health` section and update it 
-with interval equal to `KUMA_DP_SERVER_HDS_REFRESH_INTERVAL`. Alternatively, it's possible to omit a `serviceProbe` section and develop custom
-automation that periodically updates the `health` of the inbound.
-
-Comparing to `HealthCheck` policy `serviceProbes` have some advantages:
-- knowledge about health is propagated back to `kuma-cp` and could be seen both in Kuma GUI and Kuma CLI
-- scalable with thousands of Dataplanes
+If there is a `serviceProbe` configured for the inbound, Kuma will automatically fill the `inbound.health` section and update it 
+with interval equal to `KUMA_DP_SERVER_HDS_REFRESH_INTERVAL`.
+Alternatively, it's possible to omit a `serviceProbe` section and develop custom automation that periodically updates the `health` of the inbound.
 
-but at the same time unlike `HealthCheck` policy `serviceProbes`:
-- works only when `kuma-cp` is up and running
-- can't check TLS between Envoys
+If grpc stream with Envoy is disconnected then Kuma considers this proxy as offline.
+It will however, still advertise inbounds as their latest health was.
```

Comment:
```suggestion
It will, however, still advertise inbounds using the final update on their health before disconnection.
```

If I understood correctly.